catalogue/webapp/components/WorksSearchResult/WorksSearchResult.tsx

```diff
@@ -80,8 +67,13 @@ const WorkSearchResult: FunctionComponent<Props> = ({
   resultPosition,
 }: Props) => {
   const productionDates = getProductionDates(work);
-  const workTypeIcon = getWorkTypeIcon(work);
   const archiveLabels = getArchiveLabels(work);
+  const cardLabels = [
+    { url: null, text: work?.workType?.label },
```

Comment:
as far as I can tell [`workType` isn't `nullable`](https://github.com/wellcomecollection/wellcomecollection.org/blob/master/common/model/catalogue.ts#L9)

I can't find a rule to not allow null checking on non-optional fields, but it would be nice.